.travis.yml

```diff
@@ -15,7 +15,6 @@ os:
   - windows
 
 node_js:
-  - "10"
```

Comment:
`10` will pick the latest 10 version, which currently is `v10.20.1` (see [here for example](https://travis-ci.com/github/adobe/node-cloud-blobstore-wrapper/jobs/340636577)). I think that's still useful to know in case we update our images to the latest node 10.x version and to check if something broken since 10.19.